tests/phpunit/api/v3/MailingTest.php

```diff
@@ -410,8 +410,10 @@ public function testMailerSendTest_email() {
     $mail = $this->callAPISuccess('mailing', 'create', $this->_params);
 
     $params = array('mailing_id' => $mail['id'], 'test_email' => 'alice@example.org', 'test_group' => NULL);
+    // Per https://lab.civicrm.org/dev/core/issues/229 ensure this is not passed through!
+    $params['id'] = $mail['id'];
```

Comment:
@eileenmcnaughton how has this test been passing is it just that there isn't a difference in ids between the civicrm_mailing table the civicrm_mailing_job table here?

Reply:
@seamuslee001 I was able to replicate it when I added to the test so it wasn't transposing 'mailing_id' to 'id' in the api wrapper (I haven't fully thought that through) - 

in the api 'id' refers to mailing_id - but is passed to MailingJob as if it were MailingJob id - hence the bug

Reply:
(mailing id is supposed to be required here & it makes sense it should be)

Reply:
hmm interesting. So the test was succeeding in that it was setting a mailing_id but no id but once the id got set then it started failing is that the case?